consult-project-extra.el

```diff
@@ -58,31 +69,52 @@
   "Compute the project files given the ROOT."
   (setq root (file-name-as-directory root))
   (let* ((project (consult-project-extra--project-with-root root))
-         (project-files-relative-names t)
+         (current-project (project-current))
+         (project-files-relative-names (and current-project
+                                            (string= root (project-root current-project))))
```

Comment:
note: `project-root` is new as of Emacs 28.1. I'd run this through `package-lint`.

Reply:
(bringing this up because the requirements at the top declare Emacs 27.1)